src/components/Footer/Footer.module.css

```diff
@@ -0,0 +1,83 @@
+.wrapper {
+    padding-bottom: 24px;
+    margin-top: 64px;
+    max-width: 186px;
+    margin-left: 16px;
+    min-height: 310px;
+    
+}
+
+.list {
+    padding: 0;
+    list-style-type: none;
+}
+
+.footerItem {
+    color: white;
+    margin-bottom: 32px;
+    font-family: "InterRegular";
+    font-size: 24px;
+    line-height: 24px;
+}
+
+.copyright {
+    margin-top: 48px;
+    color:white;
+    font-family: "InterRegular";
+    font-size: 14px;
+    line-height: 14px;
+}
+
+.link, .link:visited {
+    text-decoration: none;
+    color: white;
+}
+
+.link:hover {
+    text-decoration: underline;
+}
+
+@media (min-width: 768px) {
+    .wrapper {
+        display: flex;
+        flex-direction: row;
+        margin: auto;
+        padding: 0 0 24px 0;
+        max-width: 720px;
+        min-height: 0;
+        justify-content: space-between;
+    }
+
+    .list {
+        display: flex;
+        flex-direction: row;
+        max-height: 18px;
+    }
+    .footerItem {
+        font-size: 16px;
+        line-height: 18px;
+ 
+    }
+    .footerItem:not(:last-child) {
+        margin-right: 18px;
+    }
+
+    .copyright {
+        font-size: 14px;
+        line-height: 16px;
+        margin: auto 0;
+    }
+}
+
+@media (min-width: 1024px) {
+    .wrapper {
+        max-width: 928px;
+    }
+}
+
+@media (min-width: 1280px) {
+    .wrapper {
+        max-width: 1184px;
```

Comment:
you need `max-width` only on the widthest screens for this element

Reply:
sorry, I don't understand. without `max-width` on every screen, the width will be incorrect for all of them.

Reply:
remove max-width for all screens but not for the biggest

Reply:
![image](https://user-images.githubusercontent.com/110398466/212477293-92aca09a-bca6-4b87-9ccc-c53dd55e6074.png)
this is what happens on 1024

Reply:
Yeah, I wrote the comment in the worng way. My bad.

I meant that it looks good, until we check what is happening between brakpoints and the "paddings" are not dynamic. So we can leave it for now because this fix should be applied for the rest of the desctions as well.

_____________


Otherwise right now it looks like it does on the screenshot and should be fixed

Reply:
so is it better to do percentages instead of max-width?

Reply:
No no, let's leave the current solution, but fix it for 320 screens and medium screens: the space around footer and also there is a horisontal scroll on 320 screen width.